### PR TITLE
[APM] Add a bit more details in the network troubleshooting doc

### PR DESCRIPTION
**Pull request by pierotibou**
### What does this PR do?
Adds a bit more details in the network troubleshooting doc. It aims at guiding the users to troubleshoot more easily connectivity issues in containerized environments.

### Motivation
We had a customer call where the users had an egress rule that prevented them to connect the tracer to the agent.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/troubleshooting/connection_errors.md [hunk 1]
```diff
@@ -192,11 +192,16 @@ If you cannot isolate the root cause, [contact Datadog Support][4] with:
 
 In containerized setups, submitting traces to `localhost` or `127.0.0.1` is often incorrect since the Datadog Agent is also containerized and located elsewhere. **Note**: AWS ECS on Fargate and AWS EKS on Fargate are exceptions to this rule.
 
-Determine if the networking between the application and the Datadog Agent matches what is needed for that configuration. 
+Determine if the networking between the application and the Datadog Agent matches what is needed for that configuration.
 
-In particular, check to make sure that the Datadog Agent has access to port `8126` and that your application is able to direct traces to the Datadog Agent’s location. 
+In particular, check to make sure that the Datadog Agent has access to port `8126` (or to the port you have defined) and that your application is able to direct traces to the Datadog Agent’s location.
+To do so, you can run the following command, from the application container and by replacing the `{agent_ip}`, `{agent_port}` variables:
```
Comment (hestonhoffman) [inline content/en/tracing/troubleshooting/connection_errors.md:198]:
```suggestion
To do so, you can run the following command from the application container, replacing the `{agent_ip}` and `{agent_port}` variables:
```

File: content/en/tracing/troubleshooting/connection_errors.md [hunk 2]
```diff
@@ -192,11 +192,16 @@ If you cannot isolate the root cause, [contact Datadog Support][4] with:
...
 
-A great place to get started is the [APM in-app setup documentation][6]. 
+`curl -X GET http://{agent_ip}:{agent_port}/info`.
```
Comment (hestonhoffman) [inline content/en/tracing/troubleshooting/connection_errors.md:200]:
~~~suggestion
```
curl -X GET http://{agent_ip}:{agent_port}/info
```
~~~

File: content/en/tracing/troubleshooting/connection_errors.md [hunk 3]
```diff
@@ -192,11 +192,16 @@ If you cannot isolate the root cause, [contact Datadog Support][4] with:
...
+
+If this command fails, your container cannot access the agent. Refer to the following sections to get hints on what could be causing this issue.
```
Comment (hestonhoffman) [inline content/en/tracing/troubleshooting/connection_errors.md:202]:
```suggestion
If this command fails, your container cannot access the Agent. Refer to the following sections to get hints on what could be causing this issue.
```

File: content/en/tracing/troubleshooting/connection_errors.md [hunk 4]
```diff
@@ -211,7 +216,7 @@ See the table below for example setups. Some require setting up additional netwo
 | [AWS EKS on Fargate][9] | Do not set `DD_AGENT_HOST` |
 | [AWS Elastic Beanstalk - Single Container][10] | Gateway IP (usually `172.17.0.1`) |
 | [AWS Elastic Beanstalk - Multiple Containers][11] | Link pointing to the Datadog Agent container name |
-| [Kubernetes][12] | 1) [Unix Domain Socket][13], 2) [`status.hostIP`][14] added manually, or 3) through the [Admission Controller][15] |
+| [Kubernetes][12] | 1) [Unix Domain Socket][13], 2) [`status.hostIP`][14] added manually, or 3) through the [Admission Controller][15]. Also, if using TCP, don't forget to check the [network policies][21] applied on your container |
```
Comment (hestonhoffman) [inline content/en/tracing/troubleshooting/connection_errors.md:219]:
```suggestion
| [Kubernetes][12] | 1) [Unix Domain Socket][13], 2) [`status.hostIP`][14] added manually, or 3) through the [Admission Controller][15]. If using TCP, check the [network policies][21] applied on your container |
```

**Review by hestonhoffman — CHANGES_REQUESTED** [review]

**Comment by pierotibou** [comment]
Thanks @hestonhoffman I've just applied your suggestions

**Pull request merged**